subsys/bluetooth/host/hci_core.c

```diff
@@ -432,7 +432,8 @@ int bt_hci_cmd_send_sync(uint16_t opcode, struct net_buf *buf,
 
 	status = cmd(buf)->status;
 	if (status) {
-		LOG_WRN("opcode 0x%04x status 0x%02x", opcode, status);
+		LOG_WRN("opcode 0x%04x status %s(0x%02x)", opcode,
+			bt_hci_err_to_str(status), status);
```

Comment:
In https://github.com/zephyrproject-rtos/zephyr/pull/75442 the formatting is `%u %s`. Should we use the same formatting here?

Reply:
Yes, I agree. We should should change it to that format